bin/pasta_evaluate_patches.py

```diff
@@ -240,6 +240,7 @@ def dump_characteristics(repo, characteristics, ignored, relevant, filename):
 
                 row = {'id': message_id,
                        'from': mail_from,
+                       'from_name': c.mail_from[0],
```

Comment:
Guess that should be refactored to avoid confusion.
from -> from_email.

Reply:
And have a look at line 231. We can simplify it to sth. like:

from_email, from_name = c.mail_from.